packages/extension/app/newtab.html

```diff
@@ -13,11 +13,11 @@ <h1>Cypress is currently automating this browser.</h1>
     <p>Opening new tabs may interfere with tests and cause failures.</p>
     <p>Please note:</p>
     <ul>
-      <li>Any opened tabs will be closed when Cypress is stopped.</li>
+      <li>Any opened tabs will be closed between tests.</li>
       <li>Tests currently running may fail while another tab has focus.</li>
```

Comment:
Is this still true if we set focus before each command? Or it could still fail if the user changed focus during the command.

Reply:
We've made it less likely a user can effectively use a new tab while a test is running, but I think it's still true that if they do open one in the middle of a long-running command, it could interfere with that command's execution.